packages/remix-dev/cli/commands.ts

```diff
@@ -208,7 +208,7 @@ export async function watch(
       ? remixRootOrConfig
       : await readConfig(remixRootOrConfig);
 
-  devServer.liveReload(config);
+  void devServer.liveReload(config);
```

Comment:
Out of curiosity, why not `await` here? _(I'm not very familiar with the surrounding code)_

Reply:
The `await` was moved to the never-resolving `Promise` on the next line in #5223. I'm guessing it's to keep the process running until the `exitHook(() => clean(config))` [in `liveReload.ts`](https://github.com/remix-run/remix/blob/d8bcdc1b209d64cba4769ede49043f59ea22b0dc/packages/remix-dev/devServer/liveReload.ts#L73) has time to finish.

Some of the other floating-promises are in non-async contexts. Others I don't know... is it better to mark the current way it is used or to make them `await` and then possibly face an previously resolved issue?

Reply:
Cool, thanks for the context. And this is really exciting work btw! 🙌